esteid_certificates/constants.py

```diff
@@ -0,0 +1,55 @@
+    "EE Certification Centre Root CA": "EE_Certification_Centre_Root_CA.pem",
+    "SK ID Solutions EID-Q 2024R": "EID_Q_2024R.pem",
+    "SK ID Solutions EID-Q 2024E": "EID_Q_2024E.pem",
+    "SK ID Solutions EID-NQ 2021E": "EID_NQ_2021E.pem",
+    "SK ID Solutions EID-Q 2021E": "EID_Q_2021E.pem",
+    "SK ID Solutions EID-NQ 2021R": "EID_NQ_2021R.pem",
+    "SK ID Solutions ORG 2021E": "ORG_2021E.pem",
+    "SK ID Solutions EID-Q 2021R": "EID_Q_2021R.pem",
+    "SK ID Solutions ORG 2021R": "ORG_2021R.pem",
+    "ESTEID2018": "esteid2018.pem",
+    "EID-SK 2016": "EID-SK_2016.pem",
+    "NQ-SK 2016": "NQ-SK_2016.pem",
+    "KLASS3-SK 2016": "KLASS3-SK_2016_EECCRCA_SHA384.pem",
+    "ESTEID-SK 2015": "ESTEID-SK_2015.pem",
+    "SK TSA CA 2023E": "SK_TSA_CA_2023E.pem",
+    "SK TSA CA 2023R": "SK_TSA_CA_2023R.pem",
+    "SK TIMESTAMPING UNIT 2024E": "SK_TIMESTAMPING_UNIT_2024E.pem",
+    "SK TIMESTAMPING UNIT 2024R": "SK_TIMESTAMPING_UNIT_2024R.pem",
+    "SK TIMESTAMPING AUTHORITY 2023": "SK_TIMESTAMPING_AUTHORITY_2023.pem",
+    "SK TIMESTAMPING AUTHORITY 2022": "SK_TIMESTAMPING_AUTHORITY_2022.pem",
+    "SK TIMESTAMPING AUTHORITY 2021": "SK_TIMESTAMPING_AUTHORITY_2021.pem",
+    "SK TIMESTAMPING AUTHORITY 2020": "SK_TIMESTAMPING_AUTHORITY_2020.pem",
+    "TEST of EE Certification Centre Root CA": "TEST_of_EE_Certification_Centre_Root_CA.pem",
+}
+LIVE_CERTS = {
+    "EE Certification Centre Root CA": "EE_Certification_Centre_Root_CA.pem",
+    "TEST of EE-GovCA2018": "TEST_of_EE-GovCA2018.pem",
+    "TEST of EE Certification Centre Root CA": "TEST_of_EE_Certification_Centre_Root_CA.pem",
+    "TEST of SK ID Solutions ROOT G1E": "TEST_SK_ROOT_G1_2021E.pem",
+    "TEST of SK ID Solutions ROOT G1R": "TEST_SK_ROOT_G1_2021R.pem",
+    "TEST of ESTEID2018": "TEST_of_ESTEID2018.pem",
+    "TEST of ESTEID-SK 2015": "TEST_of_ESTEID-SK_2015.pem",
+    "TEST of EID-SK 2016": "TEST of EID-SK 2016_reissued.pem",
+    "TEST of NQ-SK 2016": "TEST of NQ-SK 2016_reissued.pem",
+    "TEST of KLASS3-SK 2016": "TEST_of_KLASS3-SK_2016.pem",
+    "TEST of SK ID Solutions EID-NQ 2021E": "TEST_EID-NQ_2021E.pem",
+    "TEST of SK ID Solutions EID-NQ 2021R": "TEST_EID-NQ_2021R.pem",
+    "TEST of SK ID Solutions EID-Q 2021E": "TEST_EID-Q_2021E.pem",
+    "TEST of SK ID Solutions EID-Q 2021R": "TEST_EID-Q_2021R.pem",
+    "TEST of SK ID Solutions EID-Q 2024E": "TEST_of_SK_ID_Solutions_EID-Q_2024E.pem",
+    "TEST of SK ID Solutions EID-Q 2024R": "TEST_of_SK_ID_Solutions_EID-Q_2024R.pem",
+    "TEST of SK ID Solutions ORG 2021E": "TEST_ORG_2021E.pem",
+    "TEST of SK ID Solutions ORG 2021R": "TEST_ORG_2021R.pem",
+    "DEMO SK TIMESTAMPING AUTHORITY 2020": "DEMO_SK_TIMESTAMPING_AUTHORITY_2020.pem",
+    "TEST of SK TSA CA 2023E": "TEST_of_SK_TSA_CA_2023E.pem",
+    "TEST of SK TSA CA 2023R": "TEST_of_SK_TSA_CA_2023R.pem",
+    "DEMO SK TIMESTAMPING AUTHORITY 2023E": "demo_tsu_ecc_2023.pem",
+    "DEMO SK TIMESTAMPING AUTHORITY 2023R": "demo_tsu_rsa_2023.pem",
+}
```

Comment:
issue: Now this is odd. I'm guessing they should be the other way around? Currently the TEST and DEMO certs are in the `LIVE_CERTS`.